tensorrt_llm/_torch/models/modeling_llava_next.py

```diff
@@ -416,6 +415,43 @@ def pack_image_features(self,
                                     device=image_features[0].device)
         return new_image_features, feature_lens
 
+    def _pad_for_batching(
```

Comment:
What is the benefit of aligning the `num_patch` dimension to `max_patches` within a batch? It seems we stack all image features along `num_patch` dimension later at L480-L483